include/zephyr/kernel.h

```diff
@@ -6576,6 +6576,16 @@ int k_thread_runtime_stats_enable(k_tid_t thread);
  */
 int k_thread_runtime_stats_disable(k_tid_t thread);
 
+/**
+ * @brief Check if runtime statistics gathering is enabled for a thread
+ *
+ * This routine checks whether the specified thread has enabled runtime statistics.
+ *
+ * @param thread ID of thread
+ * @return true if usage statistics are enabled for the given thread, otherwise false
+ */
+bool k_thread_runtime_stats_is_enabled(k_tid_t thread);
```

Comment:
If this just depends on the thread struct, that header appears in the include paths already.  You can make this inline without worry and save a few code bytes for users.